adafruit_funhouse/network.py

```diff
@@ -77,10 +77,19 @@ def init_io_mqtt(self):
                 "Adafruit IO secrets are kept in secrets.py, please add them there!\n\n"
             ) from KeyError
 
-        return self.init_mqtt(IO_MQTT_BROKER, 1883, aio_username, aio_key, True)
+        return self.init_mqtt(
+            IO_MQTT_BROKER, MQTT.MQTT_TLS_PORT, aio_username, aio_key, True
+        )
 
     # pylint: disable=too-many-arguments
-    def init_mqtt(self, broker, port=1883, username=None, password=None, use_io=False):
+    def init_mqtt(
+        self,
+        broker,
+        port=MQTT.MQTT_TLS_PORT,
```

Comment:
I advise using a numeric '8883' instead of the TLS_PORT const, which reads nicer and less confusing to new users (don't have to dig into MQTT if they're trying to set a port for mosquito or home assistant). 

Rest looks OK!

Reply:
Ok, thanks.